packages/@aws-cdk/cicd/lib/pipeline-deploy-stack-action.ts

```diff
@@ -0,0 +1,84 @@
+
+import cfn = require('@aws-cdk/aws-cloudformation');
+import api = require('@aws-cdk/aws-codepipeline-api');
+import cdk = require('@aws-cdk/cdk');
+import cxapi = require('@aws-cdk/cx-api');
+
+export interface PipelineDeployStackActionProps {
+  /**
+   * The CDK stack to be deployed.
+   */
+  stack: cdk.Stack;
+  /**
+   * The CodePipeline stage in which to perform the deployment.
+   */
+  stage: api.IStage;
+  /**
+   * The CodePipeline artifact that holds the synthesized app.
+   */
+  inputArtifact: api.Artifact;
+
+  /**
+   * The name to use when creating a ChangeSet for the stack.
+   * @default CDK-CodePipeline-ChangeSet
+   */
+  changeSetName?: string;
+  /**
+   * The runOrder for the CodePipeline action creating the ChangeSet.
+   * @default 1
+   */
+  createChangeSetRunOrder?: number;
+  /**
+   * The runOrder for the CodePipeline action executing the ChangeSet.
+   * @default ``createChangeSetRunOrder + 1``
```

Comment:
I think by convention we leave an empty line before `@default`.

Reply:
Ok.